pkg/network/setup/netpod/netpod.go

```diff
@@ -308,6 +308,9 @@ func (n NetPod) composeDesiredSpec(currentStatus *nmstate.Status) (*nmstate.Spec
 					return nil, fmt.Errorf("pod link (%s) is missing", podIfaceName)
 				}
 				ifacesSpec, err = n.managedTapSpec(podIfaceName, ifIndex, podIfaceStatusByName)
+				if nmstate.AnyInterface(ifacesSpec, hasIP4GlobalUnicast) {
+					spec.LinuxStack.IPv4.ArpIgnore = pointer.P(procsys.ARPReplyMode1)
+				}
```

Comment:
We should probably do this on a per-interface and not per netns.

However, this can be explored in a follow up to improve even the exiting core bridge binding.